### DISCUSSION TITLE
Using this as main part of an application

### DISCUSSION THREAD

**Discussion by Flmmng** [alelievr/NodeGraphProcessor#113] [General]
Hi,

Just wondering if it would be possible to use the NodeGrapProcessor as the main part of a business application? Or will it only work within the Unity editor?

What I'm looking to do is create an application that allow the user to build a graph and then have the application generate output in a specific format based on the graph.

Thanks!

**Comment by alelievr** [comment]
Hello,

NodeGrapProcessor is built on top of the [GraphView](https://docs.unity3d.com/ScriptReference/Experimental.GraphView.GraphView.html) API which is Editor only. So you won't be able to edit the graph at runtime.

Only the processing and graph modification using the API is possible at runtime. If you want to edit the graph visually at runtime you'll need to use a custom solution.
